packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -507,7 +507,12 @@ module.exports = ({ types: t }) => {
       },
 
       Function: {
-        exit(path) {
+        exit(path, state) {
+          const multiPass = state.opts.multiPass === false ? false : true;
+          if (!multiPass) {
+            return;
+          }
+
           earlyReturnTransform(path);
 
           if (!path.node[shouldRevisit]) {
```

Comment:
Doesn't `multipass` affect this ?

It throws an error for me - `Container is falsy`. I suppose this happens when calling `path.replaceWith` on the path whose node is modified.